src/ts/components/seekbar.ts

```diff
@@ -60,6 +60,13 @@ export interface SeekBarConfig extends ComponentConfig {
    * Used to enable/disable seek preview
    */
   enableSeekPreview?: boolean;
+
+  /**
+   * Enables a fallback mode for handling ARIA details of the seekbar for platforms that don't support the 
+   * `aria-valuetext` attribute, such as Samsung Tizen. In this mode, the `aria-label` attribute is used in addition
+   * to provide the current time.
```

Comment:
nit: it's not 100% clear IMO, maybe we could reprase it like this?
```suggestion
   * `aria-valuetext` attribute, such as Samsung Tizen. In this mode, the `aria-label` attribute is used to also
   * provide the current time.
```

Reply:
I've made an attempt to clarify this in 919bce2423100e248459fc7694f9201930ce6a59. Let me know what you think.